README.md

```diff
@@ -35,13 +36,20 @@ async function main (id) {
     const render(view(await res.json()), document.body)
   } catch (e) {
     if (e instanceof HttpError) {
-      if (e.code === 401) logout()
-      if (e.code === 404) notFound()
+      if (e.type === HttpStatusCodes.unauthorized) logout()
+      if (e.type === HttpStatusCodes.not-found) notFound()
```

Comment:
We can't use `param-case` here. `camelCase` seems better.

Reply:
I agree.